localization/ekf_localizer/src/ekf_localizer.cpp

```diff
@@ -397,6 +397,15 @@ void EKFLocalizer::initEKF()
   ekf_.init(X, P, params_.extend_state_step);
 }
 
+void warnIfDelayTimeLassThanZero(const Warning & warning, const double delay_time)
```

Comment:
Please fix the typo
(LassThan -> LessThan)

Reply:
Also, how about `warnIfPoseDelayTimeLessThanZero`, since the function is explicitely for the pose time stamp?

Reply:
Fixed in [c7a9e4d](https://github.com/autowarefoundation/autoware.universe/pull/2642/commits/c7a9e4d16e98d0431bb827b9b8ac0c5164726e5a)

I can add the same function for twist timestamp or share the same function between pose and twist, but in another PR